src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -367,62 +387,28 @@ private IEnumerable<PackageSource> LoadNuGetSources(PackageId packageId, Package
                 }
             }
 
-            if (!packageSourceLocation?.SourceFeedOverrides.Any() ?? true)
-            {
-                if (!defaultSources.Any())
-                {
-                    throw new NuGetPackageInstallerException("No NuGet sources are defined or enabled");
-                }
-
-                return defaultSources;
-            }
-
-            List<PackageSource> customSources = new();
-            foreach (string source in packageSourceLocation?.SourceFeedOverrides)
-            {
-                if (string.IsNullOrWhiteSpace(source))
-                {
-                    continue;
-                }
-
-                PackageSource packageSource = new(source);
-                if (packageSource.TrySourceAsUri == null)
-                {
-                    _verboseLogger.LogWarning(string.Format(
-                        LocalizableStrings.FailedToLoadNuGetSource,
-                        source));
-                    continue;
-                }
-
-                customSources.Add(packageSource);
-            }
+            return defaultSources;
+        }
 
-            IEnumerable<PackageSource> retrievedSources;
-            if (packageSourceLocation != null && packageSourceLocation.SourceFeedOverrides.Any())
-            {
-                retrievedSources = customSources;
-            }
-            else
-            {
-                retrievedSources = defaultSources;
-            }
+        private IEnumerable<PackageSource> LoadNuGetSources(PackageId packageId, PackageSourceLocation packageSourceLocation = null, PackageSourceMapping packageSourceMapping = null)
+        {
+            var sources = packageSourceLocation != null && packageSourceLocation.SourceFeedOverrides.Any() ?
```

Comment:
nit: Purely subjective thing. I don't remember if the parens are needed or not.

```suggestion
            var sources = (packageSourceLocation?.SourceFeedOverrides.Any() ?? false) ?
```

Reply:
Not needed, but I don't have a strong preference